test/test_utils/__init__.py

```diff
@@ -949,8 +949,10 @@ def get_canary_default_tag_py3_version(framework, version):
             return "py38"
 
     if framework == "pytorch" or framework == "huggingface_pytorch":
-        if Version(version) >= Version("1.9"):
+        if Version("1.9") <= Version(version) < Version("1.13"):
```

Comment:
nit: `SpecifierSet` might better serve the need here, but this is simple enough that straightforward `Version` comparisons are also good.

Reply:
sounds good, agree but just keeping consistent with the code above.